app/Http/Transformers/PostTransformer.php

```diff
@@ -27,6 +27,7 @@ public function transform(Post $post)
                 ],
                 'caption' => $post->content->caption,
                 'status' => $post->content->status,
+                'total_reactions' => count($post->content->reactions),
```

Comment:
Oh, nice! 🏆 And correct, you'll need to either do the `withCount` or any eager-loads when making the initial Eloquent query. Let me know if you want to pair on that!